js/block.js

```diff
@@ -5599,6 +5603,12 @@ function Block(protoblock, blocks, overrideName) {
         var oldValue = this.value;
         var newValue = this.label.value;
 
+        if(newValue < 1)
+        {
+          errorMsg(_("Note value must be greater than 0"));
```

Comment:
Maybe it would be better to just to prevent the value from going below zero and not show an error message. A value of zero is OK as it will trigger an error message later.

Reply:
Done.